examples/create_rosa_cluster/create_rosa_sts_cluster/main.tf

```diff
@@ -100,11 +98,11 @@ resource "ocm_cluster" "rosa_cluster" {
   sts = local.sts_roles
 }
 
-module sts_roles {
-    source  = "rh-mobb/rosa-sts-roles/aws"
-    create_account_roles = false
-    clusters = [{
-        id = ocm_cluster.rosa_cluster.id
-        operator_role_prefix = var.operator_role_prefix
-    }]
+module operator_roles {
+    source  = "https://github.com/bardielle/terraform-provider-ocm/tree/new_resource_rosa_classic/provider/ocm_cluster_rosa_classic/iam_roles_module"
```

Comment:
Make sure to change it to the origin github path (`github.com/openshify-online/...`) before merge